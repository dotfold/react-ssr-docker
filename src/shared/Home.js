import React from 'react'

export default () => <h1>react ssr docker</h1>
