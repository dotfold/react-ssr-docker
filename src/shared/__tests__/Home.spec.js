import React from 'react'
import renderer from 'react-test-renderer'
import Home from '../Home'

describe('<Home />', () => {
  it('should render correctly', () => {
    const r = renderer.create(
      <Home />
    )
    expect(r.toJSON()).toMatchSnapshot()
  })
})
