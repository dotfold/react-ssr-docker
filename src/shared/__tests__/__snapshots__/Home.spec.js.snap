// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render correctly 1`] = `
<h1>
  react ssr docker
</h1>
`;
